Documentation/Signoffs/Localization Subsystem.md

```diff
@@ -0,0 +1,105 @@
+
+Trilateration can be used to find the coordinates of an object based on its distance relative to three beacons [2]:
+
+$(x- x_{1})^{2}+(y- y_{1})^{2} = r_{1}{}^{2}$
+
+$(x- x_{2})^{2}+(y- y_{2})^{2} = r_{2}{}^{2}$
+
+$(x- x_{3})^{2}+(y- y_{3})^{2} = r_{3}{}^{2}$
+
+In order to find the distance from the device to a given beacon this Two Way Ranging(TWR) equation is used, where c is the speed of light d is the distance from the given beacon, RR is the Received Response time, Poll Sent time, SR is the Sent Response time, RP is the Received Poll time, RF is the Received Final time, SF is the Sent Final time and ToF is the Time of Flight [3]:
+
+$ToF=\frac{(RR-SP)-(SR-RP) +(RF-SR)-(SF-RR)}{4}$
+
+These equations work by calculating the time between messages and using the speed of light to determine the distance between beacons. The coordinates are calculated by the ESP32 and sent from the ESP32 to the Raspberry PI via a wired connection using I2C protocol.
+
+Based on the article "Performance Comparison between Decawave DW1000 and DW3000 in low-power double side ranging applications"[4] and “Design of  localization System Based on Ultra-Wideband and Long Range Wire-less,”[5] if the UWB module is configured to use channel 9, the horizontal error after trilateration is 15 cm.
+
+### Turtlebot:
+
+#### LIDAR:
+LiDar sensors collect depth information about the environment using a laser. A laser is pulsed from the sensor and photons are reflected back to the sensor [6]. This allows the sensor to know the distance between itself and the surface from which the photons were reflected. The sensor spins and collects many points in order to gather depth information surrounding the sensor. The depth data gathered by the LiDar sensor is sent to the Raspberry PI and then to the OpenCR 1.0 via a usb connection. The OpenCR 1.0 will then take the depth data and process it to send back to the Raspberry PI via a usb connection.
+
+#### SLAM and ROS:
+As the AuR moves, data points are collected by the LiDar sensor and processed by a SLAM algorithm built into the ROS2, which is on board the TurtleBot3 [1]. The SLAM algorithm takes data points from the LiDar and creates a model of the environment [7]. The SLAM algorithm uses incoming points to estimate the location of the TurtleBot3 by comparing the value of the points to their expected values and making corrections as needed. This process is described by the probability:
+
+$P(x_k, m|z0_k, u0_k, x_0)$
+
+Accuracy of SLAM is dependant on the quality of the LiDar sensor but can have error as low as "5 cm given that LiDar sensor depth accuracy already has 2-3 cm error" [8]. The type of SLAM algorithm used will also impact the accuracy of the model. This information means the constraints of being within 15 cm of the path and 1m of the destination, when used in conjunction with the UWB, can be fulfilled.
```

Comment:
This does not say why the 15 cm or how it will be achieved.

Say the 15 cm is based on the literature that is referenced. Using the literature, explain how the 15 cm was achieved in the literature so it can be replicated for the project.